.gitignore

```diff
@@ -153,6 +153,8 @@ yarn-debug.log*
 yarn-error.log*
 *.swp
 db_password.txt
+admin_db_password.txt
```

Comment:
This file name might be confusing, since there's already a file for the admin db password, you should rename so it's clear that it's for super-user privileges